tests/storage-disks-vm

```diff
@@ -211,14 +154,14 @@ if lxc exec v1 -- mount | grep -qwF /boot/efi; then
   lxc exec v1 -- mokutil --sb-state | grep -Fx "SecureBoot disabled"
 fi
 
-if hasNeededAPIExtension disk_io_bus; then
-  # Remove disk device restrictions and add a NVMe disk
-  lxc stop -f v1
-  lxc config device remove v1 d1
-  lxc project unset restricted restricted.devices.disk
-  lxc project unset restricted restricted.devices.disk.paths
-  lxc project unset restricted restricted
+# Remove disk device restrictions
+lxc stop -f v1
+lxc config device remove v1 d1
+lxc project unset restricted restricted.devices.disk
+lxc project unset restricted restricted.devices.disk.paths
```

Comment:
Why do we need to do this (your commit message didnt explain the why, only the what)?

Reply:
Why did the previous form of the tests not require this stuff to be turned off?

Reply:
this became a requirement after adding new long name tests in the previous commit (this time with a block device that uses `loopdev`). Please feel free to suggest alternative ways to perform this tests if you would like.

Reply:
yes, I just figured it made sense to keep them together as they are unrelated to the restriction tests